<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeploymentToolWindow</class>
 <widget class="QMainWindow" name="DeploymentToolWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>487</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>415</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Deployment Tool</string>
  </property>
  <widget class="QWidget" name="MainContentWidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QWidget" name="deploymentWdiget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="handleWidth">
          <number>5</number>
         </property>
         <widget class="QWidget" name="deployOptionsWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="generalOptionsWidget" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>153</height>
              </size>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="platformLabel">
                <property name="text">
                 <string>Hardware Platform</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="platformComboBox">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;What hardware platform to deploy the game and assets to&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="currentText">
                 <string notr="true">Android</string>
                </property>
                <property name="maxCount">
                 <number>10</number>
                </property>
                <item>
                 <property name="text">
                  <string>Android</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="selectBuildConfigLabel">
                <property name="text">
                 <string>Select Build Configuration</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="buildConfigComboBox">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the build configuration type to deploy to the hardware.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <item>
                 <property name="text">
                  <string>Debug</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Profile</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Release</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="selectCompilerLabel">
                <property name="text">
                 <string>Select Compiler</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="compilerComboBox">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the compiler to use to build the game with&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <item>
                 <property name="text">
                  <string>Clang</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>GCC</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="fileTransferSettingLabel">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>File Transfer Option</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="fileTransferComboBox">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select how files should be deployed to the hardware - either push the files or enable the Virtual File System.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <item>
                 <property name="text">
                  <string>Push Files</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Use Virtual File System</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="deployOptionsLabel">
                <property name="text">
                 <string>Deploy Options</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QCheckBox" name="buildGameCheckBox">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Should the deploy tool plugin build the game before deployment?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Build Game</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QPushButton" name="advanceOptionsButton">
                <property name="text">
                 <string>Advanced Options</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>Icons/group_closed.png</normaloff>
                  <normalon>Icons/group_open.png</normalon>Icons/group_closed.png</iconset>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="advancedOptionsWidget" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>150</height>
              </size>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <item row="0" column="1">
               <widget class="QCheckBox" name="clearDeviceCheckBox">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Removes all game assets from the device and uninstalls the game before deployment.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Clear Device Before Deploying</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QCheckBox" name="installExecCheckBox">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Install the game executable as part of the deployment step?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Install the Executable</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <layout class="QHBoxLayout" name="vfsRemoteIpHorizontalLayout">
                <item>
                 <widget class="QLineEdit" name="vfsRemoteIpTextField">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;IP Address of the AssetProcessor to connect to. For future use.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="inputMask">
                   <string>000.000.000.000</string>
                  </property>
                  <property name="text">
                   <string>127.0.0.1</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="vfsPortLabel">
                  <property name="text">
                   <string>Port</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="vfsRemotePortTextField">
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;IP Port of the AssetProcessor to connect to. Changing this will require the AssetProcessor to be restarted.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="inputMask">
                   <string>00000</string>
                  </property>
                  <property name="text">
                   <string>45643</string>
                  </property>
                  <property name="maxLength">
                   <number>5</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="vfsRemoteIpAddressLabel">
                <property name="text">
                 <string>VFS Remote IP</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="shaderCompilerSettingsLabel">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Shader Compiler Settings</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QCheckBox" name="shaderCompilerUseAssetProcessorCheckBox">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Force shader compile requests to go through the AssetProcessor connection instead of connecting directly to the remote shader compiler.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Connect through Asset Processor</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="shaderComiplerIpLabel">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Shader Compiler IP</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <layout class="QHBoxLayout" name="shaderCompilerIpHorizonalLayout">
                <item>
                 <widget class="QLineEdit" name="shaderIpAddressTextField">
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;IP Address of the Remote Shader Compiler to connect to if not using the AssetProcessor for shader compile request.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="inputMask">
                   <string>000.000.000.000</string>
                  </property>
                  <property name="text">
                   <string>127.0.0.1</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="shaderPortLabel">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Port:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="shaderPortTextField">
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;IP port of the Remote Shader Compiler to connect to if not using the AssetProcessor for shader compile request.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="inputMask">
                   <string>00000</string>
                  </property>
                  <property name="text">
                   <string>61453</string>
                  </property>
                  <property name="maxLength">
                   <number>5</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="shaderHorizontalSpacerBeforePortLabel">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="deployWidget" native="true">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>32</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="topMargin">
               <number>4</number>
              </property>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>269</width>
                  <height>13</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="deploySpinnerLabel">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deployButton">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Deploy the game and assets to a device&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Deploy</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="deploymentOutputWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="deploymentConsoleFrame">
             <property name="minimumSize">
              <size>
               <width>395</width>
               <height>33</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>33</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="deploymentConsoleLabel">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <property name="text">
                 <string>Deployment Console</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="outputTextEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOn</enum>
             </property>
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
